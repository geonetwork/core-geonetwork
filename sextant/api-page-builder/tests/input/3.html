<!DOCTYPE html>
<html dir="ltr" lang="fr-FR">
    <head>
        <meta charset="utf-8" />
        <base href="https://www.odatis-ocean.fr/test/" />
        <title>Catalogue complet</title>
        <meta
            name="description"
            content="Acc&egrave;s direct au catalogue de donn&eacute;es ODATIS"
        />
        <link
            rel="stylesheet"
            type="text/css"
            href="/typo3temp/assets/css/d42b6e1bdf.css?1592314446"
            media="all"
        />
        <script
            type="text/javascript"
            src="/typo3temp/assets/vhs-assets-jquery.js?1594384554"
        ></script>
    </head>
    <body>
        <a id="top"></a>
        <div class="headline">
            <h2>
                <a href="/activites-1">Activit&eacute;s</a>
            </h2>
        </div>
        <a href="https://www.odatis-ocean.fr/" class="navbar-brand-footer">
            <img
                alt="Odatis Ocean"
                src="/fileadmin/user_upload/Logo-Odatis_fullsize.png"
                width="1408"
                height="579"
            />
        </a>
        <li>
            <a
                title="R&eacute;alisation : Totem num&eacute;rique"
                target="_blank"
                href="https://www.totemnumerique.com/"
                >R&eacute;alisation : Totem num&eacute;rique</a
            >
        </li>
    </body>
</html>
