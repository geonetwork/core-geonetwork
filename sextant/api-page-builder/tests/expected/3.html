<!DOCTYPE html>
<html dir="ltr" lang="fr-FR">
 <head>
  <meta charset="utf-8"/>
  <base href="/test/"/>
  <title>
   Catalogue complet
  </title>
  <meta content="Accès direct au catalogue de données ODATIS" name="description"/>
  <link href="http://mysite.org/typo3temp/assets/css/d42b6e1bdf.css?1592314446" media="all" rel="stylesheet" type="text/css"/>
  <script src="http://mysite.org/typo3temp/assets/vhs-assets-jquery.js?1594384554" type="text/javascript">
  </script>
 </head>
 <body>
  <a id="top">
  </a>
  <div class="headline">
   <h2>
    <a href="http://mysite.org/activites-1">
     Activités
    </a>
   </h2>
  </div>
  <a class="navbar-brand-footer" href="https://www.odatis-ocean.fr/">
   <img alt="Odatis Ocean" height="579" src="http://mysite.org/fileadmin/user_upload/Logo-Odatis_fullsize.png" width="1408"/>
  </a>
  <li>
   <a href="https://www.totemnumerique.com/" target="_blank" title="Réalisation : Totem numérique">
    Réalisation : Totem numérique
   </a>
  </li>
 </body>
</html>
